### ISSUE TITLE
ClassicEditor creation fails due to Invalid regular expression in the code

### ISSUE THREAD

**Issue by yulia-ponomaryova** [ckeditor/ckeditor5#16662]
## 📝 Provide detailed reproduction steps (if any)

1. Importing **CKEditor 5**, version **42.0.0** 

`<script src="https://cdn.ckeditor.com/ckeditor5/42.0.0/ckeditor5.js"></script>`

2. Calling function `ClassicEditor.create(...)`

3. Loading the page

### ✔️ Expected result

Seeing the CKEditor on the page:

![image](https://github.com/ckeditor/ckeditor5/assets/112817826/3a58b29f-8d79-4c56-ad77-fdc27df10a7e)

### ❌ Actual result

Resulting in `SyntaxError: Invalid regular expression: /([a-zA-Z0-9Ã€-Å¾]+\S?)+/gu` console error
and then `ClassicEditor.create(...)` call fails.

![image](https://github.com/ckeditor/ckeditor5/assets/112817826/6070a051-154c-4383-8b30-65202fbcbd5b)

### ❓ Possible solution

Seem like there was an update on June 26, 2024:

![Screenshot 2024-07-03 134822](https://github.com/ckeditor/ckeditor5/assets/112817826/bc92be26-b7ae-4195-8520-b7d0fe401653)

Regex should be rolled back to the previous version.

Current:  `/([a-zA-Z0-9Ã€-Å¾]+\S?)+/gu`
Previous: `/([a-zA-Z0-9À-ž]+\S?)+/gu`

## 📃 Other details

* Browser: Chrome 126.0.6478.127
* OS: Windows 11
* First affected CKEditor version: 35.4.0
* Installed CKEditor plugins: none

Planning to use CKEditor 42.0.0 for the application. And this issue is not easy to work around.

Hope it can be fixed :heart:

Thank you! Take care!

**Comment by Witoso** [comment]
Hey, thanks for the issue.

v42.0.0 introduces new installation methods, and the new CDN usage [is described in the docs](https://ckeditor.com/docs/ckeditor5/latest/getting-started/installation/quick-start.html#installing-ckeditor-5-from-cdn). I'm not sure if you used the editor before, and you're migrating, but please check the [release notes](https://github.com/ckeditor/ckeditor5/releases/tag/v42.0.0) and [migration guides](https://ckeditor.com/docs/ckeditor5/latest/updating/nim-migration/migration-to-new-installation-methods.html) for details. You're using a global that is not defined, the CDN now is written in an ESM format.

Example setup done with [Builder](https://ckeditor.com/ckeditor-5/builder/): [https://stackblitz.com/edit/xczfbb?file=index.html,main.js](https://stackblitz.com/edit/xczfbb?file=index.html,main.js)

**Comment by yulia-ponomaryova** [comment]
@Witoso Hey! Oh, my bad, I've mixed up some details. 
Updated the issue description according to version **42.0.0**. 

I am migrating from version **35.4.0** to **42.0.0**. 
However, I am facing the same issue with Regex in both versions. 

Hope this will help 😺

**Comment by Reinmar** [comment]
What you're seeing is most likely a problem with encoding. Apparently, ckeditor.js stopped being interpreted as UTF-8 file in your case. If you're experiencing this with 35.4.0 and 42.0.0 it means that it might have been triggered by the migration to a new CDN provider which happened around 26.06. However, this is extremely unlikely, because we'd be flooded with bug reports.

Are you sure that your page is using UTF-8? Maybe you changed something recently on your side.

If not, could you create a sample page on which the issue can be reproduced?

**Comment by LukaszGudel** [comment]
Hi @yulia-ponomaryova,

Could you recheck if you are still reproducing this issue on `35.4.0` version? Please, make sure that the local cache is clear and disabled so you can pull the newest version of the file from our CDN.

If the issue no longer occurs in version `35.4.0` we will apply a similar fix to all the other files.

**Comment by yulia-ponomaryova** [comment]
@LukaszGudel You are a ⭐! 

Thanks a lot! It worked like a charm for `35.4.0` ❤️ 

Looking forward to the `42.0.0` fix 🤗

**Comment by LukaszGudel** [comment]
We applied the changes to other files as well.

@yulia-ponomaryova Could you confirm that the issue on `42.0.0` version is no longer reproducible?

**Comment by yu-ponomaryova** [comment]
@LukaszGudel 
Got another issue trying to test `42.0.0`:

![image](https://github.com/ckeditor/ckeditor5/assets/153506889/f7431f6b-3f21-46cf-9746-9d2c03519b40)

I am not 100% sure, but  it seems to be coming from `ckeditor5.js` file

Importing CKEditor 5, version 42.0.0
`<script src="https://cdn.ckeditor.com/ckeditor5/42.0.0/ckeditor5.js"></script>`

Trying to troubleshoot the error, this is all I was able to discover (new version is on your right):

![image](https://github.com/ckeditor/ckeditor5/assets/153506889/d9272d1f-8c1e-4d5c-8d6d-905e229a3fb1)

Hope this helps 😺

**Comment by Witoso** [comment]
Hey, the new CDN is in ESM format so you need to use import maps and `type="module"`:

```js
	<script type="importmap">
		{
			"imports": {
				"ckeditor5": "https://cdn.ckeditor.com/ckeditor5/42.0.0/ckeditor5.js",
				"ckeditor5/": "https://cdn.ckeditor.com/ckeditor5/42.0.0/"
			}
		}
		</script>
		<script type="module">
                // your code
                </script>
```

As the default doesn't expose the global. Check the example setup: https://stackblitz.com/edit/xczfbb?file=index.html,main.js

**Comment by yu-ponomaryova** [comment]
@Witoso Nice catch! `Unexpected token export` was my bad due to not encountering ESM 🤝 

@LukaszGudel The issue is fixed in version `42.0.0`. Thanks a lot! You guys are stellar ⭐ ❤️ 😻
